### DISCUSSION TITLE
Is it possible to switch another key code?

### DISCUSSION THREAD

**Discussion by MoonChild-LDAL** [Adam777Z/xbox-controller-button-remapper#25] [Q&A]
I want to set the screenshot shortcut key to INSERT, share_key = 82, the result is 0, I found another set of key codes online that can set INSERT to 45, can we switch to this key code?

https://www.cambiaresearch.com/articles/15/javascript-char-codes-key-codes

**Comment by Adam777Z** [comment]
It was updated in version [2.0.0.0](https://github.com/Adam777Z/xbox-controller-button-remapper/releases/tag/2.0.0.0).

**Reply by MoonChild-LDAL** [reply]
share_key = 45, The result is still 0.
Please check again.
